src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -162,7 +162,7 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport with ElasticS
   }
 
   def buildAutocompleteQuery(client: TransportClient, indexname: String, criteria: LibrarySearchParams, groups: Seq[String], ontologyDAO: OntologyDAO): SearchRequestBuilder = {
-    createESAutocompleteRequest(client, indexname, createQuery(criteria, groups, ontologyDAO, searchField=fieldSuggest, phrase=true), 0, 8)
+    createESAutocompleteRequest(client, indexname, createQuery(criteria, groups, ontologyDAO, searchField=fieldSuggest, phrase=true), 0, criteria.size)
```

Comment:
This increases the default number of autosuggestions from 8 to 10, but I feel like that's an okay change to make.